tests/test_core.py

```diff
@@ -8283,12 +8290,23 @@ def test_emscripten_futexes(self):
 
   @needs_dlfcn
   @node_pthreads
-  def test_pthread_dynamic_linking(self):
+  def test_pthread_dylink_basics(self):
     self.emcc_args.append('-Wno-experimental')
     self.set_setting('PROXY_TO_PTHREAD')
     self.set_setting('EXIT_RUNTIME')
     self.do_basic_dylink_test()
 
+  @needs_dlfcn
```

Comment:
does this actually `@needs_dlfcn`?

Reply:
Yeah its a bit of misnomer.   Its basically mean support for dynamic libraries.

Reply:
Oops, fixed in a followup.